src/components/paniniCreator/formSections/formSection/formSectionTypes.d.ts

```diff
@@ -0,0 +1,34 @@
+type BaseSpecialOptionsProps = {
+  options: readonly string[];
+};
+
+type CheckboxButtonProps = BaseSpecialOptionsProps & {
+  type: "checkboxButton";
+  name: string;
+};
+
+type CheckboxProps = BaseSpecialOptionsProps & {
+  type: "checkbox";
+  name: string;
+  isValBoolean: boolean;
```

Comment:
It was meant to describe checkbox val, was it boolean or not, because checkboxes could provide values lite ["BUTTER", "HUMMUS"], not only true|false, but in future branches it got deleted, because there was no need anymore for such prop.